src/lava/magma/core/model/py/ports.py

```diff
@@ -14,31 +14,80 @@
 
 
 class AbstractPyPort(AbstractPortImplementation):
+    """Abstract class for ports implemented in python. A port must have one or
```

Comment:
python -> Python (isn't that a proper name?)

The fact it must have one or more csp_ports is not very useful for a person reading this for the first time. What is even a csp_port?

Say what the purpose of the port is. It provides an interface to send and receive messages sent via channels. PyPorts are the implementation of a Port in Python.

Reply:
Then say a few words on why there are multiple csp_ports in the first place.